install.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
you should be able to actually commit this file as an executable, would help to remove that step from the readme

Reply:
You are right, the install.sh was commited as an executable, just put it there to cover. Removed the command from README section.

Also added "Raspberry Pi only" to the header. I will look into modifying for non Raspberry Pi systems in the future.